src/main/app/src/app/shared/pipes/mimetypeToExtension.pipe.ts

```diff
@@ -0,0 +1,37 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { FileFormat } from "src/app/informatie-objecten/model/file-format";
+
+@Pipe({
+  name: "mimetypeToExtension",
+  standalone: true,
+})
+export class MimetypeToExtensionPipe implements PipeTransform {
+  fileFormatExtesions = {
+    [FileFormat["PDF"]]: ".pdf",
+    [FileFormat["JPEG"]]: ".jpg/.jpeg",
```

Comment:
So i now solved it like this, i do not have enough information from the backend to determine if it's a jpeg or jpg and i am wondering if it is even worth it to implement something like that if we can also just implement it like this

Reply:
Fine with me but you should really check it with the PO I think? Because this will then be shown to the end user right? `.jpg/.jpeg`?